Source/WebInspectorUI/UserInterface/Views/NavigationSidebarPanel.js

```diff
@@ -671,7 +648,7 @@ WI.NavigationSidebarPanel = class NavigationSidebarPanel extends WI.SidebarPanel
         if (!this._pendingViewStateCookie)
             return;
 
-        this._checkForStaleResourcesIfNeeded();
+        this.pruneStaleResourceTreeElements();
```

Comment:
in order to match existing behavior we should probably only do this `if (this._shouldAutoPruneStaleTopLevelResourceTreeElements)`